content/applications/general/users/delete_account.rst

```diff
@@ -0,0 +1,35 @@
+===========================
+Odoo.com Account Management
```

Comment:
Wondering—
If this doc is exclusively about deleting an Odoo account, then wouldn't we want to name it as such? Or are we planning to add more content to this doc later?

Reply:
I've renamed the doc: Deleting Odoo.com Account. Great suggestion.